Lib/fontbakery/utils.py

```diff
@@ -96,6 +96,19 @@ def parse_version_string(fb, s):
                  " version numbers in '{}'".format(s))
 
 
+def parse_version_head(fonts):
+    """Return a family's version number. Ideally, each font in the
+    family should have the same version number. If not, return the highest
+    version number."""
+    versions = []
+    if isinstance(fonts, list):
+      for font in fonts:
+        versions.append(float(font['head'].fontRevision))
+    else:
+      versions.append(float(fonts['head'].fontRevision))
+    return max(versions)
```

Comment:
Yep! That's com.google.fonts/test/014: """Make sure all font files have the same version value."""